benchmark/single-source/DataBenchmarks.swift

```diff
@@ -142,7 +163,7 @@ func sampleData(_ type: SampleKind) -> Data {
 }
 
 func benchmark_AccessBytes(_ N: Int, _ data: Data) {
-    for _ in 1...10000*N {
+    for _ in 0..<10000*N {
```

Comment:
@eeckstein another surprising performance hit, I noticed some really heavy traffic on `ClosedRange<Int>.count` (it seems O(N)?!) Changing to `Range<Int>` reduced the benchmark chatter.